### PR TITLE
Updated Introduction with more context and details

### PR DESCRIPTION
**Pull request by surendrasinghs**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,5 +19,23 @@
 
 ## Introduction
 
-This project provides open interoperable specifcations for creating decentralized skills and education networks. It is an adaptation of beckn protocol core specification with added taxonomies and sample network policies for the skills and education sector.
+The community is currently working on Discovery of Jobs, Training, .. and Proof of Work, Training, ...
+
+The Indian skill development and employment landscape is large, heterogeneous, diversified,  fragmented and complex; which is high on friction and low on outcomes. It inherently has significant challenges such as information asymmetry, high cost of trust verification, lack of discoverability, and portability. A monolithic approach to this would not serve the purpose, rather we need to adopt an ecosystem approach which facilitates innovative solutions.
```
Comment (core-wg-admin) [inline README.md:24]: This spec is not India-specific. Please remove all region-specific language from the content.

**Comment by core-wg-admin** [comment]
@rahul101001000 your inputs would be useful as well. Thanks

File: README.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 
 # Decentralized Skilling and Education Protocol
+This project provides open interoperable specifcations for creating decentralized skills and education networks. It is an adaptation of beckn protocol core specification with added taxonomies and sample network policies for the skills and education sector.
```
Comment (rahul101001000) [inline README.md:3]: 1. specifcations -> specifications
2. beckn protocol -> the [beckn protocol](https://becknprotocol.io/)

File: README.md [hunk 3]
```diff
@@ -18,5 +19,23 @@
 
 ## Introduction
 
-This project provides open interoperable specifcations for creating decentralized skills and education networks. It is an adaptation of beckn protocol core specification with added taxonomies and sample network policies for the skills and education sector.
+The community is currently working on Discovery of Jobs, Training, .. and Proof of Work, Training, ...
```
Comment (rahul101001000) [inline README.md:22]: The community is currently working towards enabling dynamic discovery of jobs, courses, trainings, mentors, in addition to facilitating issuance of proofs of experience, taking a course, attending a training, amongst others.

**Review by rahul101001000 — APPROVED** [review]

**Pull request merged**